app/Libraries/Elasticsearch/Search.php

```diff
@@ -141,6 +156,16 @@ public function response() : SearchResponse
         return $this->response;
     }
 
+    /**
+     * @return $this
+     */
+    public function searchAfter(?array $searchAfter)
```

Comment:
not used?

Reply:
for direct use in command line

Reply:
shouldn't this set `$this->params->searchAfter` then?